SPECS/flatbuffers/flatbuffers.spec

```diff
@@ -0,0 +1,167 @@
+%global debug_package %{nil}
+# The .so version is explicitly constructed from project version—search
+# CMakeLists.txt for FlatBuffers_Library_SONAME_MAJOR and
+# FlatBuffers_Library_SONAME_FULL—but we manually repeat the SOVERSION here,
+# and use the macro in the file lists, as a reminder to avoid undetected .so
+# version bumps.
```

Comment:
This seems to no longer be relevant. I think it used to be a comment to one of the removed macros.